sdk/src/main/java/com/meniga/sdk/webservices/requests/UpdateBudgetParameters.java

```diff
@@ -0,0 +1,18 @@
+package com.meniga.sdk.webservices.requests;
+
+import java.util.List;
+
+public class UpdateBudgetParameters extends QueryRequestObject {
```

Comment:
The "parameters" naming in the budget implementation has the potential to be a bit confusing because I'm not sure if the end effect is that the budget entries are renamed or some parameters are being changed. Feels like the concrete back end implementation is bleeding into the sdk - or am I misunderstanding something? Does the sdk user care about the parameters?